### PR TITLE
Sending email to new users

### PR DESCRIPTION
**Pull request by gauthamzz**

### CODE REVIEW

File: models/db.py [hunk 1]
```diff
@@ -184,7 +184,27 @@ def sanitize_fields(form):
     for site in current.SITES:
         site_handle = site.lower() + "_handle"
         form.vars[site_handle] = form.vars[site_handle].lower()
-
+#-----------------------------------------------------------------------------
+def inside_institute(form):
+        # Send mail to all users form the same college when someone registers
+    tos=[]
```
Comment (raj454raj) [inline models/db.py]: Spaces after assignment.

File: models/db.py [hunk 2]
```diff
@@ -184,7 +184,27 @@ def sanitize_fields(form):
     for site in current.SITES:
         site_handle = site.lower() + "_handle"
         form.vars[site_handle] = form.vars[site_handle].lower()
-
+#-----------------------------------------------------------------------------
+def inside_institute(form):
+        # Send mail to all users form the same college when someone registers
```
Comment (raj454raj) [inline models/db.py]: Tab it to 4 spaces.

File: models/db.py [hunk 3]
```diff
@@ -184,7 +184,27 @@ def sanitize_fields(form):
...
+Hello,
+%s from your college has just joined StopStalk
+send him a friend request now %s and improve your experince with StopStalk
+
+Regards,
+StopStalk Team
+              """ % (form.vars.first_name + " " + form.vars.last_name,
+                     URL('default', 'search'),
+                     URL("default", "unsubscribe", scheme=True, host=True, extension=False))
+    for to in tos:
```
Comment (raj454raj) [inline models/db.py]: Why loop again ?

Comment (raj454raj) [inline models/db.py]:
```
    query = (atable.institute == form.vars.institute)
    rows = db(query).select(atable.email, atable.stopstalk_handle)
    subject = "New user registered from your college"
    message = """
Hello %s,
%s from your college has just joined StopStalk
Send a friend request now %s for better experience on StopStalk
Regards,
StopStalk
              """ % (row.stopstalk_handle,
                        form.vars.first_name + " " + form.vars.last_name,
                        URL('default', 'search'),
                     URL("default", 
                             "unsubscribe", 
                             scheme=True, 
                             host=True, 
                             extension=False))    
        for row in rows:
            send_mail(to=row.email, subject=subject, message=message)
```

**Comment by raj454raj** [comment]
@gauthamzz : Do not close PRs. Its okay to have multiple commit on the same PR! We can then squash it into one commit.

**Comment by gauthamzz** [comment]
I am really sorry .I have send you a new pr already

**Pull request closed without merging**